### DISCUSSION TITLE
Migrating to Typeorm V3

### DISCUSSION THREAD

**Discussion by olivermrbl** [medusajs/medusa#2839] [Feature Requests]
### What
Migrate to Typeorm V3

**Comment by juanzgc** [comment]
Partial boilerplate that attempts to reduce as many merge conflicts (keyword attempts lol) 😇 
https://github.com/medusajs/medusa/pull/2563

**Comment by olivermrbl** [comment]
Migration to Typeorm 0.3.11 merged in #3041
